project/src/main/webapp/create-volunteering-opportunity.js

```diff
@@ -1,4 +1,19 @@
 $(function() {
   populatePrefilled('interests');
   populatePrefilled('skills');
+  setActionToCorrectRedirectURL();
 });
+
+/**
+ * Set the opportunity form action to include query parameter for eventId.
+ */
+async function setActionToCorrectRedirectURL() {
+  const params = (new URL(document.location)).searchParams;
+  const eventId = params.get('event-id');
+
+  const opportunityForm =
+    document.getElementById('volunteering-opportunity-form');
+  opportunityForm.action =
+      `/volunteering-form-handler?event-id=${eventId}`;
```

Comment:
nit: 2 spaces instead of 4

Reply:
this is when the line is too long, we go to the next line with 2 spaces - linter does not give error.